sklearn/feature_selection/from_model.py

```diff
@@ -136,12 +136,12 @@ def _get_support_mask(self):
             estimator = self.estimator_
         else:
             raise ValueError(
-                'Either fit the model before transform or set "prefit=True"'
-                ' while passing the fitted estimator to the constructor.')
+                'Either fit SelectFromModel before transform or set "prefit='
+                'True" and pass a fitted estimator to the constructor.')
         scores = _get_feature_importances(estimator, self.norm_order)
-        self.threshold_ = _calculate_threshold(estimator, scores,
-                                               self.threshold)
-        return scores >= self.threshold_
+        threshold = _calculate_threshold(self.estimator, scores,
```

Comment:
Actually, doesn't matter mostly. ``_calculate_threshold`` only checks whether ``estimator.penalty=='l1'`` in some cases. I *guess* you could come up with a case where that's true for ``estimator`` but not ``estimator_`` or vice versa, but changing ``penalty`` during fit is actually violating API constraints, right? (because no trailing underscore).
I changed it, because it's more obvious.

What did you want tested?